lib/screens/create_game2/create_game_map.dart

```diff
@@ -0,0 +1,75 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:niira/loading.dart';
+import 'package:niira/services/location_service.dart';
+import 'package:niira/utilities/map_styles/create_game_map.dart';
+import 'package:provider/provider.dart';
+
+class CreateGameMap extends StatefulWidget {
+  @override
+  State<CreateGameMap> createState() => CreateGameMapState();
+}
+
+class CreateGameMapState extends State<CreateGameMap> {
+  final Completer<GoogleMapController> _controller = Completer();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: FutureBuilder(
```

Comment:
If you add the type here eg. FutureBuilder<Location> it will kickstart the type inference so you'll get nicer type checking (and it adds a bit of documentation for free)

Reply:
ooh thank you!